junit_tests/TimeDeltaTest.java

```diff
@@ -0,0 +1,494 @@
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+import org.python.Object;
+import org.python.stdlib.datetime.TimeDelta;
+import org.python.types.Bool;
+import org.python.types.Float;
+import org.python.types.Int;
+import org.python.types.Str;
+
+import java.util.Collections;
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class TimeDeltaTest {
+
+    private TimeDelta constructTimeDelta(long days, long seconds, long microseconds) {
+        return new TimeDelta(new Object[] {Int.getInt(days), Int.getInt(seconds), Int.getInt(microseconds)}, Collections.emptyMap());
+    }
+
+    @Test
+    public void testConstructor() {
```

Comment:
You only test your constructor with kwargs and not with args. It should be valid to write timedelta(10,2) for example.